crates/rpc/rpc/src/eth/helpers/transaction.rs

```diff
@@ -41,6 +45,48 @@ where
 
         Ok(hash)
     }
+
+    async fn send_raw_transaction_sync(&self, tx: Bytes) -> Result<Receipt, Self::Error> {
```

Comment:
we need to adjust the return to 

https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc-eth-api/src/helpers/transaction.rs#L150-L150